config/development.json

```diff
@@ -31,6 +31,7 @@
 	"features": {
 		"ad-tracking": false,
 		"akismet/siteless-checkout": true,
+		"bloganuary": true,
```

Comment:
Thinking out loud - we want the bloganuary changes to enable/disable by default in the month of January.  I wonder if there is a way to handle this in the config here itself, or if we combine the bloganuary config with date checks in the JS later.

Reply:
I thought this would be a good way to start because this way we can test it before January, but maybe I should make a single check function so that we can add a date check later in one place.

Reply:
Yup its a fine way to start. Your `isBloganuary` function that everything is relying on would probably be a good place to add the date check once we are ready.